apps/ui/pages/reservation-unit/[id].tsx

```diff
@@ -765,11 +765,12 @@ const ReservationUnit = ({
         end,
       });
 
-      addReservation({
+      await addReservation({
```

Comment:
I'd prefer using `then()`, for uniformity.

Reply:
`async` `await` is prettier when there is a lot of them.
In this case it's not too bad, but in admin ui mutation code it's going to be `await` we are not gonna do
```
mut1.then((x) => {
  mut2.then((y) => {
   mut3.then(z) => {
   })
  })
})
```
So if consistency is important, `then` should be removed.